### ISSUE TITLE
No Admin Login after version upgrade possible

### ISSUE THREAD

**Issue by AlphaUnitDeveloper** [joomla/joomla-cms#44634]
### Steps to reproduce the issue

Currently, I am running Joomla 5.1.2 on a good maintained Linux System of a big German provider df.eu.
On demand, I will send you the phpinfo() output, but this has nothing to do with my problem.

In Joomla 5.1.2., all of my plugins are up to date.
If I am going from System->Update->Joomla to upgrade my version to (currently) 5.2.2., me and my colleagues cannot login 
to the administrator area any more. We cleared cache, we deleted cookies, we changed browsers.

### Expected result

Login to www.mysite.de/administrator 

### Actual result

Impossible to login to www.mysite.de/administrator. The Login procedure is not even moving to the 2FA step. 

### System information (as much as possible)

Just not logging in.

### Additional comments

The problem occured already as I tried to upgrade from Joomla 5.1.2 to 5.1.4. It's not new.

**Comment by AlphaUnitDeveloper** [comment]
Some extra information.

I stayed logged in the administrator interface in one browser window.

I installed the version 5.2.2.

I changed the administrator login password and the passwords of two other administrator accounts in the opened browser window. 
So I had definitely three current passwords from single administrator account of the website.

I tried to login from different PCs and browsers. No effect. I cannot log in after a version update. Am I stuck in 5.1.2. forever?

**Comment by Attila-SWE** [comment]
I don't think this is the place for individual troubleshooting but some suggestions for troubleshooting if you are using a non live test server where you are testing upgrade to 5.2.2.

Activate system debug and maximum php error reporting. It might guide you in the right direction.

Are all cookie settings like cookie domain correct?

One thing I would try is to disable MFA and see if you can log in then. Do you have any plugins or customizations of login or MFA?

**Comment by AlphaUnitDeveloper** [comment]
Hello Attilla,

as a software developer myself I already tried the "individual" troubleshooting ways.
I definitely have all correct cookie settings, the MFA hast nothing common with my problem (already set it on and off multiple times).  At the moment I don't have any  plugins or customizations of login or MFA.

**Comment by Attila-SWE** [comment]
Is frontend login working (with or without MFA) or is both "broken"?

**Comment by Attila-SWE** [comment]
Have you tried deleting administrator/cache/autoload_psr4.php? I have heard that the file sometimes causes mysterious things.

**Comment by brianteeman** [comment]
Are you getting an error or does it just do nothing?

Is there any error in the browser console?

Are you sure the update was successful - did you check the update log file?

**Comment by AlphaUnitDeveloper** [comment]
Hello, here are answers on all questions:

>>Is frontend login working (with or without MFA) or are both "broken"?
- The frontend is working, I have no customer Login Mask in the frontend. So only the Backend login mask is not working, without any error messages

>> Have you tried deleting administrator/cache/autoload_psr4.php? I have heard that the file sometimes causes mysterious things.
- Why should I delete some random files I never heard about? No, I haven't.

>> Are you getting an error or does it just do nothing?
- The login mask is just not letting me into the administration area.

I will check the last 2 questions later today/tomorrow.

Thank everybody for your concerns about my issue.

**Comment by brianteeman** [comment]
> Hello, here are answers on all questions:
> 
> > > Is frontend login working (with or without MFA) or are both "broken"?
> 
> * The frontend is working, I have no customer Login Mask in the frontend. So only the Backend login mask is not working, without any error messages


So you don't know if frontend log in is working as you do not have it on this site and have not tested it

> 
> > > Have you tried deleting administrator/cache/autoload_psr4.php? I have heard that the file sometimes causes mysterious things.
> 
> * Why should I delete some random files I never heard about? No, I haven't.

This file is regenerated automatically if it doesnt exist so it is perfectly safe to delete it and see if the regenerated file solves your problem. I doubt it will but there is no harm in trying


> 
> > > Are you getting an error or does it just do nothing?
> 
> * The login mask is just not letting me into the administration area.

so you get no error message?

> 
> I will check the last 2 questions later today/tomorrow.
> 
> Thank everybody for your concerns about my issue.

**Comment by Hackwar** [comment]
Any news on this one? If there is no response in the next 2 weeks, we are going to close this ticket.

**Comment by fgsw** [comment]
> Any news on this one? If there is no response in the next 2 weeks, we are going to close this ticket.

Remember @Hackwar.

**Comment by alikon** [comment]
closing for now no more  response
